src/v3.ts

```diff
@@ -46,6 +47,9 @@ export default function generateTypesV3(
   const objectsAndArrays = JSON.parse(
     JSON.stringify(propertyMapped),
     (_, node) => {
+      // avoiding runtime exceptions for `null` values
+      if (node === null) return null;
```

Comment:
If we want to remove the `null` values, I think we should return `undefined` instead–that’ll remove the node. Returning the node as-is will just keep as-is, which will likely not solve the problem.